woocommerce/woocommerce
# [CYS] Update `Hero product chessboard` pattern

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/44547



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new page or post and add the `Hero product chessboard` pattern.
2. In the editor, check that you don't see the `Write title` placeholder.
3. Check the margins between the title and the text in the left bottom square is reduced.

| Before | After |
|--------|--------|
| <img width="1111" alt="Screenshot 2024-02-12 at 16 02 39" src="https://github.com/woocommerce/woocommerce/assets/186112/7c0b4952-7a3a-4e56-a32a-b554f813b4c6">|<img width="1110" alt="Screenshot 2024-02-12 at 16 01 49" src="https://github.com/woocommerce/woocommerce/assets/186112/67931769-97d2-4d55-8858-048247f0d418">|



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

"Hero product chessboard" pattern: remove placeholder and update margins.

#### Comment 

</details>
